src/mongoose_server_api.erl

```diff
@@ -26,11 +26,12 @@ status() ->
         case lists:keysearch(mongooseim, 1, application:which_applications()) of
             false ->
                 {false, String1 ++ " MongooseIM is not running in that node.",
-                 "MongooseIM is not running in that node"};
+                 "MongooseIM is not running in that node", "MongooseIM is not running in that node"};
             {value, {_, _, Version}} ->
                 {true,
                  String1 ++ io_lib:format(" MongooseIM ~s is running in that node.", [Version]),
-                 lists:nth(1, string:split(Version, "-"))}
+                 lists:nth(1, string:tokens(Version, "-")),
+                 lists:nth(3, string:tokens(Version, "-"))}
```

Comment:
I think that there is a slight issue with a parsing here.  When I check a commit, I get a hash like `"commitHash": "gb936ad4bd"`, but it should be `b936ad4bd` without the 'g' at the start. I think the 'g' comes from `git describe` adding it. Maybe we should cut off the 'g' to make it right.

Reply:
thx for this comment